common/script/content/appearance/chair.js

```diff
@@ -0,0 +1,5 @@
+import prefill from './prefill.js';
+
+export default prefill({
+   black: {}
```

Comment:
I did color because of @lemoness statement of "(since we will probably have other colors eventually)."
Let me know